services/common/src/components/project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,472 @@
+  const project = useSelector(getProject);
+  const transformedProjectSummaryAuthorizationTypes = useSelector(
+    getTransformedProjectSummaryAuthorizationTypes
+  );
+  const dropdownProjectSummaryPermitTypes = useSelector(getDropdownProjectSummaryPermitTypes);
+  const projectSummaryAuthorizationTypesArray = useSelector(
+    getProjectSummaryAuthorizationTypesArray
+  );
+
+  const notApplicableText = "N/A";
+
+  let processedOtherActPermitResult: any[] = [];
+
+  const createStatusColumn = (text: string, badgeStatus: PresetStatusColorType) => ({
+    key: text,
+    title: "Status",
+    render: () => <Badge status={badgeStatus} text={text} />,
+  });
+
+  const createStatusBadge = (text: string, badgeStatus: PresetStatusColorType) => ({
+    status: badgeStatus,
+    text: text,
+  });
+
+  const statusColumn = {
+    key: "status",
+    title: "Status",
+    render: (record) => <Badge status={record.status.status} text={record.status.text} />,
+  };
+
+  const minesActStatusColumn = createStatusColumn("Submitted", "success");
+
+  const minesActColumns: ColumnsType<IAuthorizationSummary> = [
+    renderTextColumn("project_type", "Type", false),
+    renderTextColumn("permit_no", "Permit", false),
+    renderTextColumn("date_submitted", "Date", false),
+    minesActStatusColumn,
+  ];
+
+  const otherActColumns: ColumnsType<IAuthorizationSummary> = [
+    renderTextColumn("project_type", "Type", false),
+    renderTextColumn("permit_no", "Authorization", false),
+    renderTextColumn("ams_tracking_number", "Tracking #", false),
+    renderTextColumn("date_submitted", "Date", false),
+    statusColumn,
+  ];
+
+  const parseProjectTypeLabel = (authType: string) => {
+    const projectType = dropdownProjectSummaryPermitTypes.find((x) => x.value === authType);
+    return projectType?.label;
+  };
+
+  const parseTransformedProjectSummaryAuthorizationTypes = (
+    projectSummaryPermitType: string,
+    authType: string
+  ) => {
+    const projectType = transformedProjectSummaryAuthorizationTypes.find(
+      (x: any) => x.code === projectSummaryPermitType
+    );
+
+    const jobType = projectType?.children.find((x: any) => x.code === authType);
+    return jobType?.description;
+  };
+
+  const getPermitNumber = (permit_guid: string): string => {
+    const permit = permits.find(({ permit_guid: id }) => id === permit_guid);
+    return permit?.permit_no ?? notApplicableText;
+  };
+
+  const loadMinesActPermitData = (authorizations) => {
+    const result = authorizations
+      .filter(
+        (authorization) => authorization.project_summary_authorization_type === "MINES_ACT_PERMIT"
+      )
+      .map((authorization) => {
+        const dateSubmitted = formatDateTimeTz(authorization.ams_submission_timestamp);
+        const projectType = parseProjectTypeLabel(authorization?.project_summary_permit_type[0]);
+        const permitNo =
+          authorization?.project_summary_permit_type[0] === "AMENDMENT"
```

Comment:
Would be good to put the project summary permit type into an enum if we don't already have one

Reply:
Same here, enum has been created